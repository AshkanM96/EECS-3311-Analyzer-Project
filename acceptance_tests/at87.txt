for_all
open_set_enumeration
boolean_constant(true)
boolean_constant(true)
boolean_constant(true)
boolean_constant(true)
close_set_enumeration
type_check
evaluate
reset
for_all
open_set_enumeration
boolean_constant(true)
boolean_constant(true)
boolean_constant(false)
boolean_constant(true)
close_set_enumeration
type_check
evaluate
reset
there_exists
open_set_enumeration
boolean_constant(false)
boolean_constant(false)
boolean_constant(false)
boolean_constant(false)
close_set_enumeration
type_check
evaluate
reset
there_exists
open_set_enumeration
boolean_constant(false)
boolean_constant(false)
boolean_constant(true)
boolean_constant(false)
close_set_enumeration
type_check
evaluate
