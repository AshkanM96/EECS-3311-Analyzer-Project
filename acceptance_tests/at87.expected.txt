  Expression currently specified: ?
  Report: Expression is initialized.
->for_all
  Expression currently specified: (&& ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (&& {?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, True, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, True, True, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (&& {True, True, True, True})
  Report: OK.
->type_check
  Expression currently specified: (&& {True, True, True, True})
  Report: (&& {True, True, True, True}) is type-correct.
->evaluate
  Expression currently specified: (&& {True, True, True, True})
  Report: True
->reset
  Expression currently specified: ?
  Report: OK.
->for_all
  Expression currently specified: (&& ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (&& {?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (&& {True, True, False, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, False, True, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (&& {True, True, False, True})
  Report: OK.
->type_check
  Expression currently specified: (&& {True, True, False, True})
  Report: (&& {True, True, False, True}) is type-correct.
->evaluate
  Expression currently specified: (&& {True, True, False, True})
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->there_exists
  Expression currently specified: (|| ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (|| {?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, False, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, False, False, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (|| {False, False, False, False})
  Report: OK.
->type_check
  Expression currently specified: (|| {False, False, False, False})
  Report: (|| {False, False, False, False}) is type-correct.
->evaluate
  Expression currently specified: (|| {False, False, False, False})
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->there_exists
  Expression currently specified: (|| ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (|| {?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (|| {False, False, True, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, True, False, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (|| {False, False, True, False})
  Report: OK.
->type_check
  Expression currently specified: (|| {False, False, True, False})
  Report: (|| {False, False, True, False}) is type-correct.
->evaluate
  Expression currently specified: (|| {False, False, True, False})
  Report: True
